forge/test/mlir/test_ops.py

```diff
@@ -1904,3 +1904,82 @@ def forward(self, a, b):
     co_out = [co.to("cpu") for co in co_out]
     fw_out = [fw_out] if isinstance(fw_out, torch.Tensor) else fw_out
     assert all([compare_with_golden_pcc(golden=fo, calculated=co, pcc=0.99) for fo, co in zip(fw_out, co_out)])
+
+
+@pytest.mark.xfail(
+    reason="RuntimeError: Found Unsupported operations while lowering from TTForge to TTIR in forward graph - repeat"
```

Comment:
Make sure to open issues on MLIR side attached to the Bringup milestone with the requirement of adding these ops. Something like this:
- https://github.com/tenstorrent/tt-mlir/issues/1063